fendermint/app/src/cmd/objects.rs

```diff
@@ -442,16 +442,76 @@ async fn handle_object_upload(
         })
     })?;
 
+    let hash_seq_hash = tag_entangled_data(&iroh, &ent, &metadata_hash)
```

Comment:
shouldn't the entangler tag it's own data? ie, it could return some list of temp tags (UUID / similar) instead of relying on the auto tag and having to scan all tags below? I think we need to avoid any design that does a full scan of tags.